starter_code/components.js

```diff
@@ -0,0 +1,32 @@
+class AwesomePictureThing extends React.Component {
+    
+    constructor(){
+        super();
+        this.state = {show: false}
+    }
+    _toggle(){
```

Comment:
Is there a shorter way you could have done this? Maybe a one-liner using the `!` operator?

...OK I'll show you cuz I want to:

```js
this.setState({ show: !this.state.show });
```

So this line of code relies on the fact that `this.state.show` is a boolean, so by putting a `!` in front of it we can flip it to the exact opposite (if it was `true` it becomes `false` and vice versa); so no matter what, you switch the state to the opposite of what it was.